### PR TITLE
Fixed schemas and added row transform function

### PR DESCRIPTION
**Pull request by Joe-Heffer-MPE**
This has been tested (all schemas) with target-csv.

### CODE REVIEW

**Comment by cmerrick** [comment]
Hi @Joe-Heffer-MPE, thanks for your contribution!

In order for us to evaluate and accept your PR, we ask that you [sign a contribution license agreement](https://stitch-cla-enforcer.herokuapp.com/). It's all electronic and will take just minutes.

**Comment by cmerrick** [comment]
You did it @Joe-Heffer-MPE!

Thank you for signing the Singer Contribution License Agreement.

**Comment by Joe-Heffer-MPE** [comment]
Is there any feedback on this pull request?

Thanks

File: setup.py [hunk 1]
```diff
@@ -6,13 +6,13 @@
...
+    url="https://github.com/singer-io/tap-gemini",
     classifiers=["Programming Language :: Python :: 3 :: Only"],
     py_modules=["tap_gemini"],
     install_requires=[
-        "singer-python=5.4.1",
-        "requests=2.21.0",
-        "pytz=2018.4",
+        "singer-python",
+        "requests",
+        "pytz",
```
Comment (luandy64) [inline setup.py:15]:
```suggestion
        "singer-python==5.4.1",
        "requests==2.21.0",
        "pytz==2018.4",
```

Comment (luandy64) [reply setup.py:15]: We want to keep the pinned versions, but the `=` is a syntax error

File: tap_gemini/__init__.py [hunk 2]
```diff
@@ -3,70 +3,60 @@
...
+    # Build timezone-aware datetime object
+    return datetime.datetime.combine(
+        date=date,
+        time=datetime.time(0, tzinfo=pytz.UTC)
+    )
+
+
+def today() -> datetime.datetime:
+    """Current calendar date"""
+    return cast_date_to_datetime(date=datetime.date.today())
```
Comment (luandy64) [inline tap_gemini/__init__.py:59]: This function seems redundant. You could just have
```
def cast_date_to_datetime(date: datetime.date = datetime.date.today()) -> datetime.datetime:
```

**Review by luandy64 — CHANGES_REQUESTED** [review]
I'll merge this since this is still in alpha, but I think the `setup.py` changes need to be made first

**Comment by Joe-Heffer-MPE** [comment]
Hi, thank you for your feedback, it's greatly appreciated. I've made the changes discussed above and merged the changes onto the master branch on Joe-Heffer-MPE/tap-gemini.

**Comment by Joe-Heffer-MPE** [comment]
I've made changes and tested the full integration (i.e. full historic replication) by piping the data into target-csv using Windows and Linux environments. I've pushed the changes to master on https://bitbucket.org/Joe_Heffer/tap-gemini

**Comment by dmosorast** [comment]
Closing this, handled in #3. Thanks!

PR #3 rebased these changes onto a new branch and pulled a merge commit from bitbucket/Joe_Heffer/master.

**Pull request closed without merging**